.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,8 @@
 
 <!-- How does this look? Add a screenshot if you can -->
 
-<!-- Annotate your PR with label proper labels (architecture impacted, type of improvement, etc.)
+<!-- Annotate your PR with label proper labels (architecture impacted, type of improvement,
```

Comment:
Do you want to update .github/PULL_REQUEST_TEMPLATE.md in this PR?

Reply:
Yes, this is not even minor it's totally irrelevant,  just cosmetic. No point doing another PR